README.md

```diff
@@ -82,13 +90,21 @@ To include `core`, create an `.eslintrc` file in your project and include the fo
 }
 ```
 
-#### Additional config example
+```js
+{
+  "extends": "@springernature/eslint-config/legacy",
```

Comment:
I feel it's unclear here if this should be used in addition to the core config, or instead of it. Perhaps a line above to clarify would be helpful?

Reply:
👍